README.md

```diff
@@ -141,21 +142,46 @@ The service will respond to requests on `localhost:2424`, and the admin data wil
 
 ### Configuration
 
-Configuration is handled by [Viper](https://github.com/spf13/viper#putting-values-into-viper).
-The easiest way to set config during development is by editing the [config.yaml](./config.yaml) file.
+Configuration is handled by [Viper](https://github.com/spf13/viper#putting-values-into-viper). The easiest way to set config during development is by editing the [config.yaml](./config.yaml) file. You can also set the config through environment variables. For instance:
+
+```bash
+export PBC_COMPRESSION_TYPE="none"
+```
+##### Rate limiter configuration
+
+Prebid Cache's rate limiting feature, that has the downside of considerable memory consumption, is enabled by default for a maximum of 100 requests per second. From the [config.yaml](./config.yaml) file, use the `rate_limiter.enabled` and `rate_limiter.num_requests` options to either disable the rate limiter or modify its request capacity. For instance:
+
+```yaml
+rate_limiter:
+  enabled: false
+```
 
-You can also set the config through environment variables. For example:
+disables the rate limiter. We could also disable setting the following environment variable:
+
+```bash
+export PBC_RATE_LIMITER_ENABLED="false"
+```
+
+In contrast, we could keep the rate limiter running and set its maximum number of requests to a value other than 100. Inside [config.yaml](./config.yaml):
+
```

Comment:
The number of requests in the sentence above refers to `100` vs that in the example below refers to `150`. Can you please update either to match with the other so that they are the same value?

Reply:
They are meant to be different, but you are right, it is confusing at first glance. Reworded the sentence to clarify a bit.